tidb-cloud/manage-user-access.md

```diff
@@ -1,125 +1,141 @@
+Under this structure:
 
-In the upper-right corner of the TiDB Cloud console, click your account name and select **Logout**.
+- Billing occurs at the organization level, while retaining visibility of usage in each project and cluster.
 
-## Manage user passwords
+- You can view all members in your organization.
 
-> **Note:**
->
-> The content in this section is only applicable to TiDB Cloud registration with email and password. If you sign up for TiDB Cloud with Google or GitHub, your password is managed by Google or GitHub and you cannot change it using the TiDB Cloud console.
+- You can also view all members in your project.
 
-The password for TiDB Cloud account must meet the following requirements:
+## Manage Organization Access
 
-- The password must be at least 8 characters in length.
-- The password must contain at least 1 uppercase letter, 1 lowercase letter, and 1 number.
+### View Organizations
 
-To improve your system security, if you sign up for TiDB Cloud with email and password, it is recommended that you reset your password every 90 days.
+To check which project you belong to, perform these steps:
 
-To change the password, take the following steps:
+1. Click the account name in the upper-right corner of the TiDB Cloud console.
+
+2. Click **Organization Settings**. You can view your organization.
+
+### Switch Organizations
+
+If you are a member of multiple organizations, you can switch your account between organizations.To switch between organizations, perform these steps:
 
 1. Click the account name in the upper-right corner of the TiDB Cloud console.
 
-2. Click **Account**.
+2. Select **Switch Organization** in the drop-down menu, and click the organization you want to switch to.
 
-3. Click the **Change Password** tab.
+### Set time zone for Organization
 
-4. Click **Change Password**, and then check your mail box for the link to reset the password.
+If you are the organization owner, you can modify the system display time according to your time zone.To change the local timezone setting, perform the following steps:
 
-> **Note:**
->
-> The new password must not be the same as any of the previous four passwords.
+1. Click the account name in the upper-right corner of the TiDB Cloud console.
 
-If your password is not changed within 90 days, you will get a prompt to reset your password when you log in to TiDB Cloud. It is recommended that you follow the prompt to reset the password.
+2. Click **Organization Settings**. The organization settings page is displayed.
 
-## Manage user profiles
+3. Click **Time Zone**.
 
-In TiDB Cloud, you can easily manage your profile, including your first name, last name, and phone number.
+4. Click the drop-down list and select your time zone.
+
+5. Click **Confirm**.
+
+### Invite an organization member
+
+If you are the owner of an organization, you can invite organization members. Otherwise, skip this section.To invite a member to an organization, perform the following steps:
```

Comment:
```suggestion
If you are the owner of an organization, you can invite organization members. Otherwise, skip this section.

To invite a member to an organization, perform the following steps:
```